### ISSUE TITLE
Remote service layers not rendering

### ISSUE THREAD

**Issue by timlinux** [GeoNode/geonode#6132]
## Expected Behavior

I am trying to add a remote service layer from 

http://data.fao.org/maps/wms

to a GeoNode 3.0 instance.

I tested with QGIS and the same service works nicely. 

![image](https://user-images.githubusercontent.com/178003/83154267-d3437080-a0f7-11ea-8151-0d06b0781ce0.png)

## Actual Behavior

I used this layer for testing:

```
crs=EPSG:4326&dpiMode=7&format=image/png&layers=AQUAMAPS:cov03_42660&styles&url=http://data.fao.org/maps/wms
```

The same layer added in GeoNode shows a legend, but no map:

![image](https://user-images.githubusercontent.com/178003/83154576-2ddccc80-a0f8-11ea-869f-e52f829a9d7e.png)

If I look in the layer view and use the network inspector I can see an error for the WMS call:

![image](https://user-images.githubusercontent.com/178003/83155238-ef93dd00-a0f8-11ea-89a5-26ba1bab142a.png)

The url it is trying to access is:

http://data.fao.org/maps/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&LAYERS=geonode%3AAQUAMAPS%3Acov03_42660&STYLES=&SRS=EPSG%3A3857&CRS=EPSG%3A3857&TILED=true&ENV=&WIDTH=256&HEIGHT=256&BBOX=0%2C-20037508.342789244%2C20037508.342789244%2C0

Decoded:

http://data.fao.org/maps/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image/png&TRANSPARENT=true&LAYERS=geonode:AQUAMAPS:cov03_42660&STYLES=&SRS=EPSG:3857&CRS=EPSG:3857&TILED=true&ENV=&WIDTH=256&HEIGHT=256&BBOX=0,-20037508.342789244,20037508.342789244,0

Opening that URL in my browser gives:

![image](https://user-images.githubusercontent.com/178003/83155723-7052d900-a0f9-11ea-9214-fd758c9f291e.png)

If I compare that with the WMS calls made in QGIS:

http://data.fao.org/maps/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&BBOX=-180,-90,180,90&CRS=EPSG:4326&WIDTH=405&HEIGHT=807&LAYERS=AQUAMAPS:cov03_42660&STYLES=&FORMAT=image/png&DPI=72&MAP_RESOLUTION=72&FORMAT_OPTIONS=dpi:72&TRANSPARENT=TRUE

You can see the key difference is that the LAYERS list has prepended the word GeoNode:

Client | Layers
-------|--------
GeoNode Call | LAYERS=geonode:AQUAMAPS:cov03_42660
QGIS Call | LAYERS=AQUAMAPS:cov03_42660

If I take the URL generated by GeoNode and simply remove the geonode suffix in the layers list, the resulting URL correctly returns an image:

![image](https://user-images.githubusercontent.com/178003/83157505-96797880-a0fb-11ea-8b1c-f4d530543737.png)

## Steps to Reproduce the Problem

  1. Add the WMS resource http://data.fao.org/maps/wms
  2. Add the First AQUAMAPS layer that appears in the list
  3. Open the layer view and the map is not shown

## Specifications

  - GeoNode version: 3,0
  - Installation method: SPCGeoNode Docker
  - Platform: Ubuntu 20.4
  - Additional details:

You can view on the running instance here:

https://sadc-gip.org/layers/httpdatafaoorgmapswms:geonode:AQUAMAPS:cov03_42660

**Comment by t-book** [comment]
might be connected to: https://github.com/GeoNode/geonode/issues/5512

**Comment by afabiani** [comment]
**WARNING: The fix is available on GeoNode 3.1 only**

Fixed through https://github.com/GeoNode/geonode/commit/04d105534dd8ed2ddd0a5a41bc827e1d41bcf32d

![image](https://user-images.githubusercontent.com/1278021/86572663-8616b200-bf73-11ea-9a59-25fe6e5a2229.png)
